app/src/main/java/com/ryan/heroestopbuilds/MainActivity.java

```diff
@@ -255,25 +255,18 @@ public String prettyPrinter(ArrayList<String> popularSkills) {
         ArrayList<String> finalList = new ArrayList<>();
         String lgSpacing = String.format("%" + 3 + "s", "");
         String smSpacing = String.format("%" + 1 + "s", "");
+        int weGotAChromie = 0;
         //Check first skills, this means we got a Chromie
         if(popularSkills.contains("CompoundingAether") || popularSkills.contains("DeepBreath")
```

Comment:
Lets fuck this shit up, FAM.

Seeing that popularSkills is an ArrayList we can lean on pattern matching the strings within. This'll blow your fuckin mind.

for(String s : popularSkills) {
if(s.matches("(?i)(CompoundingAether|DeepBreath|TimewalkersPursuit|PeerIntoTheFuture)")){
isChromie_? = 1
}
}

I would assume those skill names can be shortened if no other heroes have matching skill names or combos of words